index.html

```diff
@@ -63,23 +63,221 @@ <h1 class="primary">
+                        <li class="tag">React.js</li>
+                        <li class="tag">Ruby and Rails</li>
+                        <li class="tag">RSpec</li>
+                        <li class="tag">Capybara</li>
+                        <li class="tag">Selenium</li>
+                    </ul>
+                </li>
+                <li class="card-works-inactive">
+                    <img
+                    src="assets/images/skill-icons.svg" alt="skills" />
+                  <h3 class="title">Skills</h3>
+                    <ul class="frame19-tags">
+                        <li class="tag">Database Management</li>
+                        <li class="tag">Version Control</li>
+                        <li class="tag">CLI</li>
+                        <li class="tag">Web Development</li>
+                        <li class="tag">API Design</li>
+                    </ul>
+                </li>
+            </ul>
+        </section>
+        <section id="contact" class="contact-form">
+            <h2 class="primary-text">
+                I'm always <br> interested in <br> hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch.
+            </h2>
+            <form method="post" autocomplete="off"  class="input-form">
+                 <input id="name-input"  class="full-name"  type="text"  placeholder="Full name"  name="full-name" />
+                <input id="email-input" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" type="email" placeholder="Email address" name="email"/>
+                <textarea id="textarea-input" class="details" id="details" placeholder="Hey I’d like to connect to..."  name="message" ></textarea>
+                <button id="submitBtn" name="submit" class="green-btn-submit" type="submit" title="press" /> Get in touch </button>
+                     <p class="contact-email">yesuf023@gmail.com</p>
+            </form>
         </section>
       </main>
       <footer class="footer">
+        <ul class="social-media-icons-foot">
+            <li class="twitter-icon">
+                <a href="">
+                    <img src="assets/images/twitter.svg" alt="twitter"/>
+                </a>
+            </li>
+            <li class="linkedin-icon">
+                <a href="">
+                    <img src="assets/images/Linkedin icon.svg" alt="Linkedin"/>
+                </a>
+            </li>
+            <li class="medium-icon">
+                <a href="">
+                    <img src="assets/images/medium.svg" alt="medium"/>
+                </a>
+            </li>
+            <li class="github-icon">
+                <a href="">
+                    <img src="assets/images/github.svg" alt="github" />
+                </a>
+            </li>
+            <li class="angellist-icon">
+                <a href="">
+                    <img src="assets/images/angellist.svg" alt="angellist" />
+                </a>
+            </li>
+        </ul>
```

Comment:
- [ Required ] I think it would be better if you removed the `fixed` positioning from the social media icons list in the `footer` because when scrolling it overlaps with the text on the page which makes it difficult to read the contents of the page. Please see the screenshot below and rectify this.

| Figma design | Your design |
|-------------------|------------------|
|![Screenshot from 2023-06-03 03-24-40](https://github.com/yesuf023/Portfolio/assets/16392046/e6e8e1c9-93b9-4619-b3cf-9359800e2dc7)|![Screenshot from 2023-06-03 03-18-53](https://github.com/yesuf023/Portfolio/assets/16392046/626cefa1-b0e8-49ed-b2aa-5a52135592dd)|